src/mower_logic/src/mower_logic/behaviors/DockingBehavior.cpp

```diff
@@ -174,21 +174,29 @@ std::string DockingBehavior::state_name() {
 
 Behavior *DockingBehavior::execute() {
 
+    if(!isGPSGood){
```

Comment:
should call it

Reply:
works without it so will remove for now

Reply:
ah, it gets reset when entering the instance. Need to just wait in a loop instead